dev/com.ibm.ws.webcontainer.security/resources/com/ibm/ws/webcontainer/security/resources/WebAppSecurityMessages.nlsprops

```diff
@@ -141,3 +141,7 @@ SEC_JACC_AUTHZ_FAILED.useraction=Contact your security administrator if this is
 SEC_FORM_LOGOUTEXITPAGE_INVALID=CWWKS9125E: The URL provided on the custom logoutExitPage of the Form-Logout service can not be displayed, requestURL={0}, logoutExitPage={1}. The logout was successful and the default Form-Logout page was displayed.
 SEC_FORM_LOGOUTEXITPAGE_INVALID.explanation=The Form-Logout service failed to successfully redirect to the page specified on the logoutExitPage servlet parameter. The specified URL is either an invalidly formed URL or the hostname is not specified on the com.ibm.websphere.security.logoutExitPageDomainList property.
 SEC_FORM_LOGOUTEXITPAGE_INVALID.useraction=Correct the URL specified on the logoutExitPage parameter.
+
+JWT_ALREADY_LOGGED_OUT=CWWKS9126E: Login using a JWT token failed because the token was previously logged out.
+JWT_ALREADY_LOGGED_OUT.explanation=A token that was previously logged out has been presented for login again. 
+JWT_ALREADY_LOGGED_OUT.useraction=Obtain a new JWT token, or set configuration attribute trackLoggedOutSSOCookiesEnabled to false, which is less secure. 
```

Comment:
ID edits and question below:
JWT_ALREADY_LOGGED_OUT=CWWKS9126E: Login using a JWT token failed because the token was previously logged out.
**Would they have provided this JWT previously? Wondering if they will know what "a JWT token" means here or whether we should say "the JWT token" to note a previously referenced token.**
JWT_ALREADY_LOGGED_OUT.explanation=A token that was previously logged out was resent for login authentication. 
JWT_ALREADY_LOGGED_OUT.useraction=Obtain a new JWT token, or set the trackLoggedOutSSOCookiesEnabled configuration attribute to false, which is less secure.

Reply:
Maybe. The main purpose of this code is to catch "cookie theft" where a cookie is stolen by eavesdropping and reused inappropriately, kind of like somebody stealing your car keys.  In addition to your recommendations I'll change "JWT token" and "token" to Json Web Token to make it more clear.  Please let me know if you think we need further chagnes.